css/amp-story-player-iframe.css

```diff
@@ -47,14 +47,17 @@ main {
 .story-player-iframe:nth-child(1),
 main .story-player-iframe[i-amphtml-iframe-position="0"] {
   transform: translate3d(0, 0, 0);
+  z-index: 1;
 }
 
 .story-player-iframe:nth-child(2),
 .story-player-iframe:nth-child(3),
 main .story-player-iframe[i-amphtml-iframe-position="1"] {
   transform: translate3d(100%, 0, 0);
+  z-index: -1;
```

Comment:
Nit: this should not be needed if the [position=0] has a z-index of 1
(it's always easier to use less z-indexes than more)